### PR TITLE
[Docs](inverted index) update docs for build index

### PR DESCRIPTION
**Pull request by Tanya-W**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Tanya-W** [comment]
run buildall

**Comment by Tanya-W** [comment]
run buildall

**Review by qidaye — DISMISSED** [review]
LGTM

File: docs/en/docs/data-table/index/inverted-index.md [hunk 1]
```diff
@@ -99,13 +99,35 @@ table_properties;
...
+```
+(**The above 'create/add index' operation needs to be executed before executing the build index**)
+
+To view the progress of the `build index`, you can use the following statement
+```sql
+show build index [FROM db_name];
+-- Example 1: Viewing the progress of all build index tasks
+show build index;
+-- Example 2: Viewing the progress of the build index task for a specified table
+show build index wehre TableName = "table1";
```
Comment (airborne12) [inline docs/en/docs/data-table/index/inverted-index.md:128]: typo

File: docs/zh-CN/docs/data-table/index/inverted-index.md [hunk 2]
```diff
@@ -97,13 +97,35 @@ table_properties;
...
+```
+(**在执行build index之前需要已经执行了以上`create/add index`的操作**)
+
+查看`build index`进展，可通过以下语句进行查看：
+```sql
+show build index [FROM db_name];
+-- 示例1，查看所有的build index任务进展
+show build index;
+-- 示例2，查看指定table的build index任务进展
+show build index wehre TableName = "table1";
```
Comment (airborne12) [inline docs/zh-CN/docs/data-table/index/inverted-index.md:126]: typo also wehre->where

**Review by airborne12 — CHANGES_REQUESTED** [review]

File: docs/en/docs/data-table/index/inverted-index.md (hunk 1 above)
Comment (Tanya-W) [reply docs/en/docs/data-table/index/inverted-index.md]: fixed

File: docs/zh-CN/docs/data-table/index/inverted-index.md (hunk 2 above)
Comment (Tanya-W) [reply docs/zh-CN/docs/data-table/index/inverted-index.md]: fixed

**Comment by Tanya-W** [comment]
run buildall

**Review by qidaye — APPROVED** [review]
LGTM

**Review by zy-kkk — APPROVED** [review]
LGTM

**Pull request merged**